src/models/ServiceLevelAgreement.ts

```diff
@@ -51,20 +54,42 @@ export class ServiceLevelAgreement implements IServiceLevelAgreement {
     this.description = serviceLevelAgreement.description
     this.enabled = serviceLevelAgreement.enabled
     this.trigger = serviceLevelAgreement.trigger
-    this.labels = serviceLevelAgreement.labels
     this.created = serviceLevelAgreement.created
     this.updated = serviceLevelAgreement.updated
     this.account = serviceLevelAgreement.account
     this.workspace = serviceLevelAgreement.workspace
     this.actor = serviceLevelAgreement.actor
     this.severity = serviceLevelAgreement.severity
+    this.type = serviceLevelAgreement.type
   }
 
   public durationInSeconds(): number {
     return this.trigger.within
   }
 
-  public getSlaType(): string {
-    return ServiceLevelAgreementType.TimeToCompletion
+  public getSlaDefinitionKeyValuePairs(): { key: string, value: unknown }[] {
```

Comment:
This is happening because we don't preserve the full state of what a user asked for in the SLA object on the backend. We turn the SLA into an automation and stick some labels on it in the db to preserve the minimal SLA state we need (severity, type, etc) but we don't have the properties exposed in the OSS client. Example:

```python
class FrequencySla:
    stale_after: timedelta
    
 flow.deploy(_sla=FrequencySla(severity="minor", stale_after=timedelta(hours=1)))
```

We store `automation.labels = {"prefect.sla.severity": "minor", "prefect.sla.type": "FrequencySla"}` in the db. So I'm "re-assembling" the interface we provided to the user at deploy time.